apps/prairielearn/src/ee/lib/validateHTML.ts

```diff
@@ -317,6 +319,76 @@ function checkIntegerInput(ast: DocumentFragment | ChildNode): ValidationResult
   };
 }
 
+/**
+ * Checks that a `pl-symbolic-input` element has valid attributes.
+ * @param ast The tree to consider, rooted at the tag to consider.
+ * @returns The list of errors for the tag, if any.
+ */
+function checkSymbolicInput(ast: DocumentFragment | ChildNode): ValidationResult {
+  const errors: string[] = [];
+  let answersName: string | null = null;
+  let usedBlank = false;
+  if ('attrs' in ast) {
+    for (const attr of ast.attrs) {
+      const key = attr.name;
+      const val = attr.value;
+      switch (key) {
+        case 'answers-name':
+          answersName = val;
+          break;
+        case 'weight':
+        case 'size':
+          assertInt('pl-symbolic-input', key, val, errors);
+          break;
+        case 'correct-answer':
+          assertFloat('pl-symbolic-input', key, val, errors);
+          if (val.match(mustacheTemplateExtractorRegex)) {
+            errors.push(
+              "pl-symbolic-input: correct-answer attribute value must not be a Mustache template. If the correct answer depends on dynamic parameters, set `data['correct_answers']` accordingly in `server.py` and remove this attribute.",
+            );
+          }
+          break;
+        case 'label':
+        case 'aria-label':
+        case 'variables':
+        case 'placeholder':
+        case 'custom-functions':
+        case 'suffix':
+          break;
+        case 'display':
+          assertInChoices('pl-symbolic-input', key, val, ['block', 'inline'], errors);
+          break;
+        case 'imaginary-unit-for-display':
+          assertInChoices('pl-symbolic-input', key, val, ['i', 'j'], errors);
+          break;
+        case 'allow-complex':
+        case 'allow-trig-functions':
+        case 'show-help-text':
+        case 'show-score':
+          assertBool('pl-symbolic-input', key, val, errors);
+          break;
+        case 'allow-blank':
+          assertBool('pl-symbolic-input', key, val, errors);
+          usedBlank = true;
+          break;
```

Comment:
I don't think this is exactly right. This assumes that the presence of `allow-blank` corresponds to it being set to `true`, but one could just as easily have `allow-blank="false"`.

Reply:
Fixed in 9b44217.

Reply:
That doesn't fix things for other elements with the same problem. I'll fix that first in another PR and then incorporate those changes here.

Reply:
Oh, yeah that would be important

Reply:
Moved to https://github.com/PrairieLearn/PrairieLearn/pull/11965.

Reply:
I resolved this in this PR in 44e13ff.